### ISSUE TITLE
loss NAN problem

### ISSUE THREAD

**Issue by hzh8311** [foamliu/InsightFace-v2#8]
I use this repo to train on my own datasets, the batch_size is set to be 4*64 and lr=0.1, then the loss becomes `nan` after servel batches say 5000, why could it be?

**Comment by foamliu** [comment]
Several approaches you can try:
1. Reduce learning rate
2. Gradient clipping
3. Normalize the data

**Comment by hzh8311** [comment]
1. the training process before loss get nan seems quite norm, the loss decrease all the time and get nan suddenly, I assume that the learning rate is the reason
2. I have gradient clipping with [-1.0, 1.0]
3. The data normalization is done by `(x-mean/std)`, should this be a problem?

**Comment by foamliu** [comment]
Maybe, please feel free to have a try, and update with your findings.
